styleguide/_layouts/default.jade

```diff
@@ -53,38 +53,38 @@ html(lang="en")
                 a(href="global/voice").navZoolander-dropdownLink Voice and Tone
 
           li.navZoolander-item
-            a(href="#").navZoolander-link.navZoolander-hasDropdown Global Elements
+            a(href="#").navZoolander-link.navZoolander-hasDropdown Global Theme
             ul.navZoolander-dropdown
               li.navZoolander-dropdownItem
-                a(href="derek/assets/").navZoolander-dropdownLink Overview
+                a(href="derek/globalTheme/").navZoolander-dropdownLink Overview
```

Comment:
This should be `global-theme` for consistency with the naming conventions for other directories `styleguide/derek`.